README.md

```diff
@@ -38,33 +38,45 @@ vendor also provided simplified solution:
-| Intel TDX | TDX MRTD/RTMR        | CC Event Log   | [Intel TDX Module 1.5 Base Architecture Specification](https://cdrdv2.intel.com/v1/dl/getContent/733575) </br> [Intel TDX Virtual Firmware Design Guide](https://cdrdv2.intel.com/v1/dl/getContent/733585) </br> [td-shim specification](https://github.com/confidential-containers/td-shim/blob/main/doc/tdshim_spec.md) |
+| Intel TDX | TDX MRTD/RTMR        | CC Event Log   | [Intel® TDX Module 1.5 Base Architecture Specification](https://cdrdv2.intel.com/v1/dl/getContent/733575) </br> [Intel® TDX Virtual Firmware Design Guide](https://cdrdv2.intel.com/v1/dl/getContent/733585) </br> [td-shim specification](https://github.com/confidential-containers/td-shim/blob/main/doc/tdshim_spec.md) |
 
 ![](docs/cc-trusted-foundation.png)
 
-## 3. SDKs
+## 3. APIs
 
-The back-end SDK or service will produce the TCB measurements:
+CC Trusted APIs aims to collect confidential primitives (i.e., measurement, event log, quote) for zero-trust design, supporting multiple deploy environments (firmware/VM/cloud native cluster).
+The [APIs](common/python/cctrusted_base/api.py) are designed to be vendor agnostic and TCG compliant APIs. The APIs will keep evolving on demand. 
+
+| API | Description  | Parameters  | Response  |
+| --- | ------------- |----- |----- |
+| get_default_algorithms | Get the default Digest algorithms supported by trusted foundation. | | The default algorithms |
+| get_measurement_count | Get the count of measurement register. | | The count of measurement registers |
+| get_measurement | Get measurement register according to given selected index and algorithms. | imr_select ([int, int]): The first is index of measurement register, the second is the algorithms ID | The count of measurement registers |
+| get_quote | Get the quote for given nonce and data. | | The `Quote` object |
+| get_eventlog | Get eventlog for given index and count. | | `TcgEventLog` object |
+
+## 4. SDKs
+
+It provides different SDKs for producing the confidential primitives in different deployment environments.
+Choose correct SDK according to your environment.
 
 | SDK | Deployment Scenarios |
 | --- | --------------- |
 | Firmware SDK | Firmware Application |
 | [VM SDK](https://github.com/cc-api/cc-trusted-api/tree/main/vmsdk) | Confidential Virtual Machine |
 | [Confidential Cloud Native Primitives (CCNP)](https://github.com/intel/confidential-cloud-native-primitives) | Confidential Cluster/Container |
 
-## 4. Examples
-
-### 4.1 Enumerate the all Integrity Measurement Register
+## 5. How to use the APIs
 
-The example code is refer to [here](/vmsdk/python/cc_imr_cli.py) as follows:
+Below example code collects measurement register count of the platform using API `get_measurement_count` using `VMSDK` in python. You can find more examples at [API usage example](docs/API-usage-example.md).
```

Comment:
Is it better to describe as -> "The example code describes the steps to fetch measurements from all integrity measurement registers using API `get_measurement_count`, `get_default_algorithms` and `get_measurement` using `VMSDK` in python."